near-sdk/src/collections/legacy_tree_map.rs

```diff
@@ -18,6 +20,10 @@ use crate::IntoStorageKey;
 /// - `above`/`below`:          O(log(N))
 /// - `range` of K elements:    O(Klog(N))
 ///
+#[deprecated(
+    since = "4.1.0",
+    note = "Use near_sdk::collections::TreeMap, if you have an existing LegacyTreeMap use LegacyTreeMap::to_tree_map to migrate"
```

Comment:
So this comment is out of date now, just remove the comment about migration and this is good